cfgov/apache/conf.d/alias.conf

```diff
@@ -8,8 +8,6 @@ Alias /financial-intuition.xml ${STATIC_PATH}/financial-intuition.xml
 # Facebook site auth
 Alias /lln8595c61g9qnvuwvtlcwo1k6kem8.html ${STATIC_PATH}/lln8595c61g9qnvuwvtlcwo1k6kem8.html
 Alias /static/f/ ${STATIC_PATH}/nemo/_/f/
-# Preserves legacy URL compatibility, but this is not actually using Wordpress
-Alias /wp-content/themes/cfpb_nemo/_/  ${STATIC_PATH}/nemo/_/
```

Comment:
This is used for https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/nemo.less#L441